src/main/java/org/jenkinsci/main/modules/instance_identity/pem/PEMHelper.java

```diff
@@ -1,5 +1,12 @@
 package org.jenkinsci.main.modules.instance_identity.pem;
 
+import org.apache.commons.codec.binary.Base64;
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+import javax.annotation.Nonnull;
+import javax.xml.bind.DatatypeConverter;
```

Comment:
IIUC the goal is to be able to eliminate this import, right? It's still used on line 188, but I think it could be replaced there with `java.util.Base64.getMimeEncoder().encodeToString()`.

Reply:
Right. Thins thing I missed